package.json

```diff
@@ -19,7 +19,6 @@
     "i18next": "^23.7.16",
     "i18next-browser-languagedetector": "^7.2.0",
     "i18next-http-backend": "^2.4.2",
-    "localforage": "^1.10.0",
```

Comment:
There is a mention on this file

https://vscode.dev/github/silversonicaxel/OpenWebScrobbler/blob/feat/unused-libs/src/localstorage.js#L2

but it is not in use anywhere, so I thought to remove it

Reply:
Makes sense. I think it was part of the project at some point, but it doesn't make sense to keep it here while it's not being used.